signatures/dilithium/dilithium-shake/param/weak.yaml

```diff
@@ -0,0 +1,11 @@
+name: weak
```

Comment:
the other schemes all include the scheme name as well here. this might make problems in presentation later on, so I'd suggest we add this here, too (applies to all)